test/e2e/README.md

```diff
@@ -95,7 +95,7 @@ The config is located in `test/e2e/config/operator.yaml`.
 Most notable ones:
 ```yaml
 variables:
-  RANCHER_VERSION: "v2.7.5" # Default rancher version to install
+  RANCHER_VERSION: "v2.9.0" # Default rancher version to install
```

Comment:
does this need an update in turtles docs pre-requisite table?

Reply:
It is in the test/e2e/README.md. This document has to contain documentation on provisioning e2e tests, and it is not maintained by the team. This change you are requesting is probably unrelated to the PR itself.

Reply:
It was just a question @Danil-Grigorev, thanks for explanation